GHKit.xcodeproj/project.pbxproj

```diff
@@ -148,8 +148,6 @@
 		00D0237F154DF43B003BAAC3 /* GHNSObject+Swizzle.h in Headers */ = {isa = PBXBuildFile; fileRef = 00D022CD154DF1C5003BAAC3 /* GHNSObject+Swizzle.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		00D02380154DF440003BAAC3 /* GHNSObject+Swizzle.m in Sources */ = {isa = PBXBuildFile; fileRef = 00D022CE154DF1C5003BAAC3 /* GHNSObject+Swizzle.m */; };
 		00D02381154DF441003BAAC3 /* GHNSObject+Swizzle.m in Sources */ = {isa = PBXBuildFile; fileRef = 00D022CE154DF1C5003BAAC3 /* GHNSObject+Swizzle.m */; };
-		00D02382154DF44D003BAAC3 /* GHNSString+UIKitUtils.h in Headers */ = {isa = PBXBuildFile; fileRef = 00D022BB154DF1C5003BAAC3 /* GHNSString+UIKitUtils.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
When I tried to build,  it said these were missing. They weren't on disk either, so I removed the proj references.